api/main.py

```diff
@@ -1,4 +1,4 @@
-from typing import Union
+from typing import Union, Iterable

```

Comment:
Let'st try out resolving comments :)
Where is Iterable being used?

Reply:
coming :)